src/component/Person.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from 'prop-types'
+
+const Person = ({ name, age = null, sex, isMarried = null, partnerName = null }) => (
```

Comment:
you could use `PropTypes.DefaultProps` instead of giving default values to the arguments